sceptre/cli/create.py

```diff
@@ -10,9 +10,15 @@
 @click.argument("path")
 @click.argument("change-set-name", required=False)
 @click.option("-y", "--yes", is_flag=True, help="Assume yes to all questions.")
+@click.option(
+    "-dr",
```

Comment:
Usually with a CLI command, `-dr` is equivalent to `-d -r`. I don't think we ought to shorten one command to two letters in the short form. Considering disabling the rollback on an action really ought to be an edge case anyway, we can just take out the short form and have the only way to do it via cli as `--disable-rollback`.

Reply:
sounds good, will do

Reply:
implemented switch `--disable-rollback/--enable-rollback`